Wikipedia/Code/WebViewController.m

```diff
@@ -290,8 +290,7 @@ - (WMFArticleFooterView*)footerLicenseView {
         @weakify(self);
         [_footerLicenseView.showLicenseButton bk_addEventHandler:^(id sender) {
             @strongify(self);
-            MWKSite* site = [[MWKSite alloc] initWithDomain:WMFDefaultSiteDomain language:@"en"];
-            [self.delegate webViewController:self didTapOnLinkForTitle:[site titleWithString:WMFLicenseTitleOnENWiki]];
```

Comment:
It's an internal page on enwp, which is not translated in to multiple languages (officially). It's true that en.wp (desktop) links to it, but mobile does not, and I think the app should use same standard and wordings as mobile.